samples/nrf9160/nrf_cloud_mqtt_multi_service/src/application.c

```diff
@@ -176,8 +181,93 @@ static void on_location_update(const struct location_event_data * const location
 	}
 }
 
+/**
+ * @brief Receives general device messages from nRF Cloud, checks if they are AT command requests,
+ * and performs them if so, transmitting the modem response back to nRF Cloud.
+ *
+ * Try sending {"appId":"MODEM", "messageType":"CMD", "data":"AT+CGMR"}
+ * in the nRF Cloud Portal Terminal card.
+ *
+ * @param msg - The device message to check.
+ */
+static char at_req_resp_buf[CONFIG_AT_HOST_CMD_MAX_LEN];
```

Comment:
Find a better place for this -- also, support direct customization, do not use `CONFIG_AT_HOST_CMD_MAX_LEN`